src/Summoner/CLI.hs

```diff
@@ -41,6 +47,35 @@ summon = execParser prsr >>= runCommand
 runCommand :: Command -> IO ()
 runCommand = \case
     New opts -> runNew opts
+    ShowInfo opts -> runShow opts
+
+runShow :: ShowOpts -> IO ()
+runShow opts =
+    case opts of
+        -- show list of all available GHC versions
+        ShowGhc          -> putText $ unlines $ map showGhcVer universe
+        ShowLicense name -> case name of
+            -- show a list of all available licenses
+            LicenseList Nothing  -> putText $ unlines $ map (\x -> show (x :: LicenseName)) universe
+            -- show a specific license
+            LicenseList (Just n) -> do
+                -- check a user`s input
+                license <- whenNothing (parseLicenseName (toText n))
+                    (fail "This wasn't a valid choice.")
```

Comment:
I guess in this case all that we need to is to `errorMessage`(so it will be red) this but also on the next line print the list of licences we support saying something like: "Here is the list of supported licenses".

Reply:
I guess it's better to pattern match here and put all the following code under the `Just` case. Because you shouldn't give any value to the license if you can not parse it. But also we don't want to see any errors. Just print the `errorMessage` and the text I mentioned above, without failing.